src/module.c

```diff
@@ -4045,6 +3957,8 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
             if (flags) (*flags) |= REDISMODULE_ARGV_NO_AOF;
         } else if (*p == 'R') {
             if (flags) (*flags) |= REDISMODULE_ARGV_NO_REPLICAS;
+        } else if (*p == '3') {
+            if (flags) (*flags) |= REDISMODULE_ARGV_RESP_3;
```

Comment:
this needs to be documented in the top comment of `RM_Call` too

Reply:
I think I added it

Reply:
I did not, added it to `moduleCreateArgvFromUserFormat`, will add it to RM_Call as well ...

Reply:
Fixed

Reply:
Only RM_ functions get their docs exported by the script...